docs/development/module/quick.md

```diff
@@ -43,12 +72,26 @@ class Egg extends QuickFormBase {
    */
   public function buildForm(array $form, FormStateInterface $form_state, string $id = NULL) {
 
-    // Egg quantity.
+    // Date+time selection field (defaults to now).
+    $form['timestamp'] = [
+      '#type' => 'datetime',
+      '#title' => $this->t('Date'),
+      '#default_value' => new DrupalDateTime(),
```

Comment:
It might be a good idea to use the current user timezone when setting the default datetime, since by default system timezone or UTC will be used, which might be different from the timezone the user has configured in their account. The same [was done](https://github.com/farmOS/farmOS/pull/665) in quick forms included in farmOS core, so it would be nice to follow these best practices also in this example.

Reply:
Good call @wotnak - I'll add that and force push the branch shortly.

Reply:
I added timezone in this commit: https://github.com/farmOS/farmOS/pull/703/commits/744fae9bf2e331e2604d2ef095d242c642dc6849

It's a bit of a shortcut because I didn't use dependency injection. I'm a bit torn on that because on the one hand it might be good to show how to do that properly, but on the other hand I really want these examples to be as simple as possible. I even considered removing the `timestamp` field from the examples entirely, but I think it's good to include.

Open to pushback on this, but that's my reasoning. :-)